src/scripts/main.js

```diff
@@ -1,3 +1,261 @@
 'use strict';
 
-// write code here
+const pageBody = document.querySelector('body');
+const tableHeader = document.querySelector('thead');
+const tableHeaderCells = tableHeader.querySelector('tr');
+const tableBody = document.querySelector('tbody');
+let isSecondClick = false;
+const toNumber = (string) => {
+  if (typeof string === 'string' && string.length !== 0) {
```

Comment:
Consider checking for null or undefined explicitly instead of relying on truthiness of 'string.length'.